generators/client/templates/angular/src/main/webapp/app/shared/user/_user.service.ts

```diff
@@ -30,12 +30,14 @@ export class UserService {
 
     constructor(private http: Http) { }
 
-    create(user: User): Observable<Response> {
-        return this.http.post(this.resourceUrl, user);
+    create(user: User): Observable<ResponseWrapper> {
```

Comment:
I need to return the user object and the response too.
So I used the ResponseWrapper here